addons/l10n_ec_website_sale/demo/website_demo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="default_website_ec" model="website">
+        <field name="name">EC Company Website</field>
+        <field name="company_id" ref="l10n_ec.demo_company_ec"/>
+        <field name="domain" model="ir.config_parameter" eval="obj().env['ir.config_parameter'].sudo().get_param('web.base.url')"/>
+    </record>
+
+    <function model="payment.provider" name="copy">
+        <value eval="[ref('payment.payment_provider_demo')]"/>
+        <value eval="{'company_id': ref('l10n_ec.demo_company_ec'), 'state': 'test'}"/>
+    </function>
+
+    <function model="payment.method" name="copy">
+        <value eval="[ref('payment_demo.payment_method_demo')]"/>
+        <value eval="{'l10n_ec_sri_payment_id': ref('l10n_ec.P19')}"/>
+    </function>
+
+    <function model="product.pricelist" name="write">
+        <value model="product.pricelist" eval="obj().sudo().search([
+                                                             ('currency_id', '=', ref('base.USD')),
+                                                             ('company_id', '=', ref('l10n_ec.demo_company_ec'))
+                                                            ]).id"/>
+        <value model="website" eval="{'sequence': 1, 'website_id': ref('l10n_ec_website_sale.default_website_ec')}"/>
+    </function>
+
+    <function model="product.product" name="write">
```

Comment:
Is this still needed?  It should happen automatically now.

Reply:
It does not seem necessary. I removed it